### PR TITLE
feat: Create Add Student Form

### PR DESCRIPTION
**Pull request by kseniacold**
####  User Story: 
* Add a button on the home page for add a new student with name, email.  Set the status code to 0.  Add a new route in App.js  

#### Change: 
* Added a button to the home page to add a new student
* Create a new route for the student addition dialog form
* Added form to add a student with validation using [constraint API ](https://developer.mozilla.org/en-US/docs/Web/API/Constraint_validation)
* When user start typing if any of the constraints are not valid - an error is displayed to the user
* Note: for now I did not add a status code, since it is automatically set on my backend, but might refactor
* For now did not connect to an actual call - will do tomorrow 
* Click on the commits for easier review 

#### Gif with Student Addition End-2-End with Validation
out

### CODE REVIEW

File: src/components/SchedList.js [hunk 1]
```diff
@@ -1,186 +1,155 @@
...
-    if (window.confirm('Are you sure you want to drop the course?')) {
-      const token = Cookies.get('XSRF-TOKEN');
-      
-      fetch(`${SERVER_URL}/schedule/${id}`,
-        {
-          method: 'DELETE',
-          headers: { 'X-XSRF-TOKEN': token }
-        })
-    .then(res => {
+    if (window.confirm("Are you sure you want to drop the course?")) {
```
Comment (wlpaulo) [inline src/components/SchedList.js:58]: Nice feature to add a confirmation. I tested the front end and was able to drop a course from the list

File: src/components/SchedList.js [hunk 2]
```diff
@@ -1,186 +1,155 @@
...
-              position: toast.POSITION.BOTTOM_LEFT
-          });
-          console.error('Post http status =' + res.status);
-        }})
-    .catch(err => {
-      toast.error("Error when adding", {
-            position: toast.POSITION.BOTTOM_LEFT
+    postResource("schedule", course, "course").then((res) => {
+      if (res.ok) {
+        toast.success("Course successfully added", {
```
Comment (wlpaulo) [inline src/components/SchedList.js:79]: Not sure if you added more css but the sliding pop-up notification looks cool!

**Review by wlpaulo — APPROVED** [review]
Nicely done Ksenia! I like the added touches with the UI features.

File: src/components/SchedList.js (hunk 2 above)
Comment (kseniacold) [reply src/components/SchedList.js:79]: That was just out-of-the-box toast notification.

**Pull request merged**